spec/views/hyrax/admin/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe 'hyrax/admin/users/index.html.erb', type: :view do
+  let(:presenter) { Hyrax::Admin::UsersPresenter.new }
+  let(:users) { [] }
+
+  before do
+    (1..11).each { |i| users << FactoryGirl.create(:user, display_name: "user#{i}", email: "email#{i}@example.com") }
```

Comment:
I will try it again... some of the changes I've made may allow it to work without persistence now. It didn't when I did my original PR but it's been long enough since then that I forget what was causing that problem.